src/Spryker/Zed/CodeGenerator/Communication/Console/BundleClientCodeGeneratorConsole.php

```diff
@@ -19,11 +19,11 @@
 class BundleClientCodeGeneratorConsole extends Console
 {
 
-    const COMMAND_NAME = 'code:generate:bundle:client';
-    const DESCRIPTION = 'Generates Client for a bundle name';
+    const COMMAND_NAME = 'code:generate:module:client';
+    const DESCRIPTION = 'Generates Client application layer for a module';
 
-    const ARGUMENT_BUNDLE = 'bundle';
-    const ARGUMENT_BUNDLE_DESCRIPTION = 'Name of the bundle (CamelCase)';
+    const ARGUMENT_BUNDLE = 'module';
```

Comment:
If someone wants to that person can also migrate all the code occurrences on top as 2nd commit.